src/log-service/templates/config.yaml

```diff
@@ -15,6 +15,9 @@ data:
   LOG_SERVICE_DISABLE_AUTH: "true"
   LOG_SERVICE_GLOBAL_TOKEN: c76e567a-b341-404d-a8dd-d9738714eb82
   LOG_SERVICE_SECRET: IC04LYMBf1lDP5oeY4hupxd4HJhLmN6azUku3xEbeE3SUx5G3ZYzhbiwVtK4i7AmqyU9OZkwB4v8E9qM
+  LOG_SERVICE_REDIS_MASTER_NAME: "harness-redis"
+  LOG_SERVICE_REDIS_SENTINEL_ADDRS: {{ include "harnesscommon.dbconnection.redisConnection" (dict "context" .Values.global.database.redis) }}
+  LOG_SERVICE_REDIS_USE_SENTINEL: {{ .Values.global.database.redis.installed  | quote }}
```

Comment:
@swapnil-harness could you confirm if LOG_SERVICE_REDIS_USE_SENTINEL when set to false, enables redis with single cluster or just enables pod-memory?
how does LOG_SERVICE_REDIS_ENDPOINT impact this condition?

Reply:
Resolved with internal discussion